content/en/serverless/installation/nodejs.md

```diff
@@ -278,6 +280,15 @@ exports.handler = async (event) => {
   return response;
 };
 ```
+Learn more about submitting custom metrics [here][3].
```

Comment:
Let's make our CTAs more natural and integrated into the copy, like on this APM page: https://docs.datadoghq.com/tracing/app_analytics/?tab=java